functions/go/apply-setters/README.md

```diff
@@ -4,12 +4,29 @@
 
 <!--mdtogo:Short-->
 
-Apply setter values on resource fields. Setters serve as parameters for template-free
-setting of field values.
+Update the field values parameterized by setters.
 
-Setters are a safer alternative to other substitution techniques which do not
-have the context of the structured data. Setters may be invoked to modify the
-package resources using this function to set desired values.
+##### Setters Definition
```

Comment:
This should just be "Definitions" and contain sections below?

Reply:
Updated.